src/test/javascript/spec/component/competencies/import/import-course-competencies.component.spec.ts

```diff
@@ -66,17 +66,19 @@ describe('ImportCourseCompetenciesComponent', () => {
         expect(component).toBeDefined();
     });
 
-    it('should initialize values correctly', () => {
-        jest.spyOn(prerequisiteService, 'getAllPrerequisitesForCourse').mockReturnValue(
-            of([
-                { id: 3, linkedCourseCompetency: { id: 11 } },
-                { id: 4, linkedCourseCompetency: { id: 12 } },
-            ] as Prerequisite[]),
-        );
-        getAllSpy.mockReturnValue(
-            of({
-                body: [{ id: 1 }, { id: 2 }],
-            } as HttpResponse<Competency[]>),
+    it('should initialize values correctly', async () => {
```

Comment:
The assertions in this test don't really seem to match how the `disabledIds` are computed. Since d8025d06d76a2d9e0c79d856f1ca82a7136b576b, we take the ID of the courseCompetency instead of the linked course competency ID for prerequisites.

That would mean that [1, 2, 3, 4] should be `disabledIds`. But is that correct?